### ISSUE TITLE
Desktop Wallet does not sync．

### ISSUE THREAD

**Issue by eikosan** [Musicoin/desktop#200]
I downloaded wallet v1.0 and sent coins from the Web Wallet to the desktop wallet.
I am troubled because the chain does not synchronize.
The balance of the desktop wallet remains "0".
Please help me.

（System: Windows10．）

**Comment by spg63** [comment]
Agreed, no sync, and rarely connects to any peers. 
macOS 10.13.3, Musicoin Wallet 1.0 according to the application header, however when going into the menu and selecting "About Musicoin-wallet" it shows at 0.0.0 (0.0.0) for version.

**Comment by jrischma** [comment]
Not sync, not connects to any peers (win7_32bit)

**Comment by Varunram** [comment]
Hey, extremely sorry for the late reply, but there's not much info on this issue that I can provide advise on. Please attach screenshots so that I may suggest something. Also, please follow the issue template next time, helps a lot. Thanks!

**Comment by jrischma** [comment]
![image](https://user-images.githubusercontent.com/11020812/37865406-baf31032-2f5a-11e8-9133-866b1620c02d.png)
now not show my wallet address with my funds.......

**Comment by Varunram** [comment]
Could you post the log file here so that I may take a look? Thanks!

**Comment by jrischma** [comment]
where is the log file¿?

**Comment by Varunram** [comment]
Advanced > Open Logs Directory

**Comment by spg63** [comment]
Just a follow up on this, after wiping the blockchain data and re-syncing the wallet everything worked as expected. I can, however, recreate the issue. If I open the musicoin wallet *after* opening a wallet for another eth-based coin, Pirl-Nautilus-Wallet in this case, the musicoin blockchain has to be wiped  again before the wallet will sync.

**Comment by Varunram** [comment]
I'm not sure about other wallets sadly, but such things should not happen with the Musicoin wallet, since we use clearly defined home directories. Maybe you should try waiting for a while to see if the chain will sync again. That being said, it might be better to open an issue with the creators of the other wallet regarding this, to see if they have something to say.

**Comment by djua** [comment]
Having the same issue, was working fine for a week and out of nowhere now Connected to: 0 peers, sometimes that 0 changes to 1 for a second and then goes back to 0 again. Tried restarting it, and starting it as administrator but nothing really helps. It's version 1.0 on windows 8.1

**Comment by jrischma** [comment]
i am testing wallet v1.01 from https://github.com/Musicoin/desktop/releases

what is the block reward for musicoin ? (250 mc or 314mc?)

thanks!

**Comment by Varunram** [comment]
@jrischma block reward is 314 MC, but the mining reward is only 250 MC (after allowances to UBI and dev fund)

**Comment by Varunram** [comment]
@djua just saw your comment, the slow peer finding rate is due to geth v2.5.3 based on which wallet 1.01 is based upon. It should find peers after a while though, if your connection is stable and you have your local clock in sync.

**Comment by jrischma** [comment]
@Varunram the block reward is 314 or 250? i see many pool with 250 but on wiki say that is 314.. what is the right reward?

**Comment by Varunram** [comment]
@jrischma As I said in my previous comment, block size is 314 MC and mining reward is 250MC.

**Comment by jrischma** [comment]
https://github.com/Musicoin/go-musicoin/wiki/The-changes-of-Musicoin-Blockchain-from-Ethereum

Coins per block: 314 MC (250 MC to miners, 50 to UBI pool according to MCIP-3，14MC to Dev pool according to MCIP-3)

UBI Pool take 50MC and Dev pool others 14MC, so on my pool the reward is 314MC, is right?

**Comment by Varunram** [comment]
Yes

**Comment by jrischma** [comment]
i set reward 314 on my pool but when my pool get a block only receive 250

![image](https://user-images.githubusercontent.com/11020812/38153713-6d77bbfc-3444-11e8-99b6-0f5299a9d5cf.png)

why?
that is right?

**Comment by Varunram** [comment]
@jrischma Please read my previous comments. The block SIZE in terms of MC is 314 MC. The mining REWARD is 250 MC, the UBI pool donation is 50MC and the dev fund donation is 14MC, bringing the total to 314 MC.

**Comment by jrischma** [comment]
@Varunram look:

"50 to UBI pool according to MCIP-3"
my pool not is "UBI pool", so why i have that send 50MC to they?

"14MC to Dev pool according to MCIP-3)" 
my pool not is "UBI pool", so why i have that send 14MC to they?

who is "UBI pool" ??

the reward is 314MC or 250MC?

**Comment by Varunram** [comment]
@jrischma I'm going to explain again. According to MCIP3, the mining reward is 250 MC, the UBI fund donation is 50MC and the dev fund donation is 14MC. Your reward as a miner is only 250MC in accordance with MCIP3. You have to send coins to the UBI fund and the dev fund because that's what the MCIP3 hardfork was all about - reducing the initial 314 MC mining reward to 250 MC.

**Comment by jrischma** [comment]
ok, so sorry, i am read bad, thanks

**Comment by jrischma** [comment]
the problem with wallet 32bit continue, not sync

**Comment by etaletai13** [comment]
Do we have any nodes coded in for the first few peers?

**Comment by jrischma** [comment]
@Varunram 
wallet on win10 64bit work fine at 100% 
wallet on win7 32bit work bad ... really bad, not sync never, freeze, close, re-open, freeze again and never, never sync

**Comment by Varunram** [comment]
@etaletai13 We download the list of initial peers from one of our servers. @jrischma not too sure why it doesn't work on Win 7, maybe @cryptofuture can can be of help since he tests the builds on Win 7.

**Comment by jrischma** [comment]
@etaletai13 please give me the list that u use, or the link for test, thanks

**Comment by etaletai13** [comment]
@jrischma It would be the same as is pulled from the servers in the app.

**Comment by Varunram** [comment]
@jrischma: https://raw.githubusercontent.com/cryptofuture/music-bootnodes/master/bootnodes.json

**Comment by cryptofuture** [comment]
Regarding https://github.com/Musicoin/desktop/issues/200#issuecomment-375895367 this happens when gmc is not running correctly. This could be due: another gmc/geth process already running before you started the wallet. Make sure no gmc or geth processes (another wallets) left before starting the wallet.
In console msg would be directory is already used. 

> Just a follow up on this, after wiping the blockchain data and re-syncing the wallet everything worked as expected. I can, however, recreate the issue. If I open the musicoin wallet after opening a wallet for another eth-based coin, Pirl-Nautilus-Wallet in this case, the musicoin blockchain has to be wiped again before the wallet will sync.

Blockchain wipe kills gmc process  before the wipe. So yes if you didn't see you accounts loading this only due another gmc or geth process was already running before your started the wallet
Actual problems with sync looks when you have dropping peers, or not peers in the status.

**Comment by jrischma** [comment]
Good News, on win7 32bit now wallet 1.01 work fine, that was solved load peers list from https://raw.githubusercontent.com/cryptofuture/music-bootnodes/master/bootnodes.json , i will continue testing on next days, thanks @Varunram and @etaletai13

**Comment by cryptofuture** [comment]
@jrischma This list should be downloaded automatically at launch (since its auto-updated) , and in case its not, wallet fallback to list file from the distribution. Might be you have problems with loading file directly, or timeout is too small for you.

**Comment by jrischma** [comment]
@cryptofuture , yes i think that should be downloaded automatically but not, so i load manually this way:
1. Advance
2. Add Peers
3. copy / paste list (all list enode:)
4. add Enodes

the message was: "Connecting 607 peers along with default remote Nodes"

now show:

Last Block: 9s 2,290,810/2,290,811
Connected to: 15 peers

before the time not run, now yes, before only connected to 2 or 3 peers, now to 15 peers

whatever, now work!

thanks for all help and support.

**Comment by etaletai13** [comment]
We could have it kill any existing gmc processes and warn about ones from other wallets, couldn't we? Or would that be unwanted behavior of some kind?

**Comment by jrischma** [comment]
in my case there was no other process gmc or geth running, only musicoin

**Comment by Varunram** [comment]
Well, I think this thread can be closed since the original issue has been solved. Anyone facing further issues with sync, please open a new issue for the same, it becomes difficult to follow these many commits, thanks!
